packages/botonic-react/src/webchat/theme/types.ts

```diff
@@ -38,29 +38,58 @@ export interface BlobProps {
   imageStyle?: any
 }
 
-export interface ThemeProps {
-  style?: any
+export interface WebchatTheme {
+  style: {
+    width?: string
+    height?: string
+    borderRadius?: string
+    fontFamily?: string
+    fontSize?: string
+    position?: 'fixed' | 'absolute'
+    right?: string
+    bottom?: string
+    backgroundColor?: string
+    boxShadow?: string
+  }
+  mobileBreakpoint: number
+  mobileStyle: {
+    width?: string
+    height?: string
+    borderRadius?: string
+    fontSize?: string
+    position?: 'fixed' | 'absolute'
+    right?: string
+    bottom?: string
+  }
+
```

Comment:
Doubt: Shouldn't the `style`, `mobileBreakpoint` and `mobileStyle` attributes be optionals?

Reply:
👍🏼 Anyway I suggest to set them as optional to keep consistency with the other attributes